### DISCUSSION TITLE
How to use ZoomPlugin when using MultiTrack

### DISCUSSION THREAD

**Discussion by ksoma105** [katspaugh/wavesurfer.js#3497] [Q&A] [answered]
Thank you for creating this very useful library.

I am using MultiTrack, and I would like to be able to zoom in and out by mouse scrolling around the cursor area, like ZoomPlugin, is it possible?

When I set the ZoomPlugin to each Track ID, it zoomed in on each Track, and I could not zoom in commonly on all Tracks.
Also, the behavior of the timeline was not correct.

Please let me know if you need additional information.

**Comment by katspaugh** [comment, marked as answer]
I’m unlikely to add additional features to multitrack unless they are paid.

**Reply by ksoma105** [reply]
Thank you for your response. I got it.
